.github/workflows/build.yml

```diff
@@ -28,8 +28,10 @@ jobs:
             shorthand: etwh
           - name: heatpump_water_heater
             shorthand: hpwh
-          - name: hvac
-            shorthand: hvac
+          - name: hvac_furnace
+            shorthand: hvac_furnace
+          - name: hvac_air_handler
+            shorthand: hvac_air_handler
```

Comment:
You need to additionally update  `.github\workflows\build-and-publish.yml` with these Matrix changes.